sql/src/test/java/io/crate/operation/projectors/ProjectionToProjectorVisitorTest.java

```diff
@@ -181,19 +189,19 @@ public void testGroupProjector() throws Exception {
         // select  race, avg(age), count(race), gender  ... group by race, gender
         List<Symbol> keys = Arrays.asList(new InputColumn(0, DataTypes.STRING), new InputColumn(2, DataTypes.STRING));
         List<Aggregation> aggregations = Arrays.asList(
-            Aggregation.finalAggregation(avgInfo, Collections.singletonList(new InputColumn(1)), Aggregation.Step.ITER),
-            Aggregation.finalAggregation(countInfo, Collections.singletonList(new InputColumn(0)), Aggregation.Step.ITER)
+            Aggregation.finalAggregation(avgInfo, Collections.singletonList(new InputColumn(1, DataTypes.DOUBLE)), Aggregation.Step.ITER),
```

Comment:
this change is not related to fetch pushdown and I think it is not needed anymore at all (even at the union branch), pls revert